web/package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "test": "jest",
     "test-watch": "jest --watch",
-    "build": "rm -rf lib && mkdir lib && tsc",
+    "build": "rm -rf lib && mkdir lib && tsc && npx babel src --out-dir lib",
```

Comment:
why `npx`? IMO it should be defined in the dev deps, otherwise it's installed globally and you can't version it

Reply:
Fixed 👍 It works, but the babel version installed is different from the one used in `scriptoni`, so there are both in the `yarn.lock`